src/unix.rs

```diff
@@ -123,13 +123,19 @@ impl Client {
             Some(w) => w,
             None => return Ok(None),
         };
-        let read = read
+        let read: c_int = read
             .parse()
             .map_err(|e| FromEnvErrorInner::CannotParse(format!("cannot parse `read` fd: {e}")))?;
-        let write = write
+        let write: c_int = write
```

Comment:
I don't think this is necessary. Did rustc type inference do its job wrong?
(Or I am wrong 😆)

Reply:
Oh. I thought, it'll look at first usage and stop. Fixed